Portal/View/UIKit/PortalMapView.swift

```diff
@@ -13,15 +13,32 @@ fileprivate let mapViewAnnotationIdentifier = "PortalMapViewAnnotation"
 
 internal final class PortalMapView: MKMapView {
     
-    fileprivate var placemarks: [MapPlacemarkAnnotation : MapPlacemark] = [:]
+    internal var placemarks: [MapPlacemark] = [] {
+        didSet {
```

Comment:
I think we should remove previous annotations

Reply:
I don't think we should have a computed property for this. I think in this case we should have an internal setter and use a private placemarks property. We should not expose the placemarks array because somebody could try to do something like

```swift
portalMapView.placemarks[1] = somePlacemark
```

which would leave the map view in an inconsistent state.